packages/l/layui.json

```
@@ -24,15 +24,15 @@
         "files": [
           "*.js",
           "css/**/*.css",
-          "font/*.@(eot|ttf|svg|woff|woff2)"
+          "font/*.@(eot|ttf|svg|woff|woff2)",
+          "*.map"
```

```suggestion
          "*.js?(.map)",
          "css/**/*.css",
          "font/*.@(eot|ttf|svg|woff|woff2)"
```
Instead of including all `map` files, I recommend editing the existing `*.js` line to also allow sourcemaps, `*.js?(.map)`